src/runtime/GlobalObjectBuiltinString.cpp

```diff
@@ -1081,6 +1081,128 @@ static Value builtinStringRaw(ExecutionState& state, Value thisValue, size_t arg
     }
 }
 
+// https://www.ecma-international.org/ecma-262/8.0/#sec-string.prototype.padstart
+// 21.1.3.14String.prototype.padStart( maxLength [ , fillString ] )
+static Value builtinStringPadStart(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+{
+    // Let O be ? RequireObjectCoercible(this value).
+    // Let S be ? ToString(O).
+    RESOLVE_THIS_BINDING_TO_STRING(S, String, padStart);
+
+    // Let intMaxLength be ? ToLength(maxLength).
+    // Let stringLength be the number of elements in S.
+    size_t intMaxLength = 0;
```

Comment:
1. You can use int64_t or uint64_t for intMaxLength.
because max value of toLength bigger than size_t in 32bit.
2. Could you use ToLength operation?